facebook/OpenBIC
# Artemis: cb: Support to monitor ACCL cable power fault

# Description
- Support to monitor ACCL power fault.
- Support to monitor ACCL cable power fault.
- Modify ACCL 3V3/12V/3V3 AUX register mapping bit.

# Motivation
- Support to monitor ACCL power fault.
- Support to monitor ACCL cable power fault.

# Test Plan
- Build code: Pass
- Check ACCL power fault event: Pass

# Log
root@bmc-oob:~# log-util all --print
2018 Mar 11 01:37:57 log-util: User cleared all logs ......
0    all      2018-03-11 03:22:24    pldmd            State Sensor: CB_SENSOR_ACCL_POWER_CABLE_1, no power good
0    all      2018-03-11 03:22:24    pldmd            State Sensor: CB_SENSOR_ACCL_POWER_CABLE_2, power good fail
0    all      2018-03-11 03:22:24    pldmd            State Sensor: CB_SENSOR_ACCL_3, no power good (3V3 power fault)
0    all      2018-03-11 03:22:24    pldmd            State Sensor: CB_SENSOR_ACCL_4, no power good (12V power fault)
0    all      2018-03-11 03:22:24    pldmd            State Sensor: CB_SENSOR_ACCL_5, no power good (3V3 aux fault)
0    all      2018-03-11 03:22:24    pldmd            State Sensor: CB_SENSOR_ACCL_6, power good fail
......
0    all      2018-03-11 01:56:14    pldmd            State Sensor: CB_SENSOR_ACCL_POWER_CABLE_8, power good fault
0    all      2018-03-11 01:56:16    pldmd            State Sensor: CB_SENSOR_ACCL_9, power good fail (3V3 power fault)
0    all      2018-03-11 01:56:17    pldmd            State Sensor: CB_SENSOR_ACCL_10, power good fail (12V power fault)
0    all      2018-03-11 01:56:18    pldmd            State Sensor: CB_SENSOR_ACCL_11, power good fail (3V3 aux fault)
